sdk/changeanalysis/Azure.ResourceManager.ChangeAnalysis/tests/Scenario/ChangeAnalysisTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ChangeAnalysis.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ChangeAnalysis.Tests.Scenario
+{
+    internal class ChangeAnalysisTests : ChangeAnalysisManagementTestBase
+    {
+        private DateTimeOffset EndTime;
+        private DateTimeOffset StartTime;
```

Comment:
Please follow the naming convention, private field should use camelCasing and prefix with `_`.